src/main/java/io/pinecone/clients/Pinecone.java

```diff
@@ -13,21 +13,25 @@
 import org.openapitools.client.model.*;
 
 import java.util.Arrays;
-import java.util.List;
-import java.util.Objects;
 import java.util.concurrent.ConcurrentHashMap;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
 public class Pinecone {
 
+    private static final ConcurrentHashMap<String, PineconeConnection> connectionsMap = new ConcurrentHashMap<>();
     private final ManageIndexesApi manageIndexesApi;
     private final PineconeConfig config;
-    private static final ConcurrentHashMap<String, PineconeConnection> connectionsMap = new ConcurrentHashMap<>();
 
     Pinecone(PineconeConfig config, ManageIndexesApi manageIndexesApi) {
         this.config = config;
         this.manageIndexesApi = manageIndexesApi;
     }
 
+    static void closeConnection(String indexName) {
```

Comment:
Just lints; ignore

Reply:
We order functions by usage, so the important ones are at the top and close() is not something we expect end users to consume.

Reply:
Gotcha, I've put it back down at the bottom